### PR TITLE
Kenneth/program page

### LINKED ISSUE #8: Create ProgramPage

### PR DESCRIPTION
**Pull request by kvsteele**
FIX #8

### CODE REVIEW

File: app/assets/javascripts/components/program_page.es6.jsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
+class ProgramPage extends React.Component {
+
+  get styles() {
+    return {
+      bodycontent: {
+      	padding: '0 70px',
+        backgroundColor: StyleConstants.colors.white,
+      },
+
```
Comment (sonia-y) [inline app/assets/javascripts/components/program_page.es6.jsx]: remove spacing between these style objects!

File: app/assets/javascripts/components/program_page.es6.jsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      topic: {
+      	color: StyleConstants.colors.orange,
+      },
+
+      textintopic: {
+      	color: StyleConstants.colors.grey,
+      }
```
Comment (sonia-y) [inline app/assets/javascripts/components/program_page.es6.jsx]: add trailing comma

File: app/assets/javascripts/components/program_page.es6.jsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    };
+  }
+
+  render () {
+    return (
+      <div style={StyleConstants.pages.default}>
+        <Header />
+
```
Comment (sonia-y) [inline app/assets/javascripts/components/program_page.es6.jsx]: remove unnecessary spaces

File: app/assets/javascripts/components/program_page.es6.jsx [hunk 4]
```diff
@@ -0,0 +1,101 @@
+class ProgramPage extends React.Component {
+
+  get styles() {
+    return {
+      bodycontent: {
```
Comment (sonia-y) [inline app/assets/javascripts/components/program_page.es6.jsx]: can we call this container

File: app/assets/javascripts/components/program_page.es6.jsx [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+      	padding: '0 70px',
+        backgroundColor: StyleConstants.colors.white,
+      },
+
+      topic: {
+      	color: StyleConstants.colors.orange,
+      },
+
+      textintopic: {
```
Comment (sonia-y) [inline app/assets/javascripts/components/program_page.es6.jsx]: rename to just text

File: app/assets/javascripts/components/program_page.es6.jsx [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+			        enriched the local community via public performances for audiences 
+			        including HCR ManorCare, the Santa Clara Valley Blind Center, and the 
+			        J.W. House of Kaiser Permanente. 
+			        </p>
+		</div>
+        <Footer />
+      </div>
+    );
+  }
+}
```
Comment (sonia-y) [inline app/assets/javascripts/components/program_page.es6.jsx]: add new line after end of file!

File: app/controllers/static_pages_controller.rb [hunk 7]
```diff
@@ -1,4 +1,8 @@
 class StaticPagesController < ApplicationController
   def home
   end
+
+  def program
+  end
 end
+
```
Comment (sonia-y) [inline app/controllers/static_pages_controller.rb]: can you add a new line for me :)

**Comment by kvsteele** [comment]
screen shot 2016-09-26 at 3 51 06 pm

**Pull request merged**